app/code/Magento/Analytics/README.md

```diff
@@ -1,18 +1,28 @@
-# Magento_Analytics Module
+# Magento_Analytics module
 
 The Magento_Analytics module integrates your Magento instance with the [Magento Business Intelligence (MBI)](https://magento.com/products/business-intelligence) to use [Advanced Reporting](https://devdocs.magento.com/guides/v2.3/advanced-reporting/modules.html) functionality.
 
 The module implements the following functionality:
 
-* enabling subscription to the MBI and automatic re-subscription
-* changing the base URL with the same MBI account remained
-* declaring the configuration schemas for report data collection
-* collecting the Magento instance data as reports for the MBI
-* introducing API that provides the collected data
-* extending Magento configuration with the module parameters:
-    * subscription status (enabled/disabled)
-    * industry (a business area in which the instance website works)
-    * time of data collection (time of the day when the module collects data)
+- Enabling subscription to the MBI and automatic re-subscription
+- Changing the base URL with the same MBI account remained
```

Comment:
This is unclear. What do you mean?

Reply:
This was already there, it was a line introduced in https://github.com/magento/magento2/commit/bfc86089837665f9c61222f8fc5ee29ba3fe1262 by @olisenko

Do you know this user or how to contact him?
Maybe hw could explain hat this line means

Reply:
@diazwatson , @jeff-matthews -  @olysenko our former employee, I'm not sure that we can receive the answer regarding this line. Probably, we may remove this line if it's confusing

Reply:
Thanks for the info @sidolov. We should definitely remove it if we can't clarify the meaning. I'm guessing it means that you can change the base URL of your Magento store without compromising the connection between the store and an existing MBI account that's been connected to the Magento app. Pinging @erikmarr (Data Solutions Technical Writer) to see if he can get us a quick answer on this one?

Reply:
Thanks @sidolov 👍 
Lets see what @olysenko says

Reply:
@diazwatson , let's remove confusing lines and proceed with delivery

Reply:
Understood!